oauth for okta clarity and $ (#4892)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md

## About OAuth 2.0 for Okta API endpoints

Most Okta API endpoints require you to include an API token with your request. Currently, this API token takes the form of an SSWS token that you generate in the Admin Console. With OAuth for Okta, you're able to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints. The scopes within the access token control that ability.

> > **Important:** Request an access token from your Okta [org authorization server](/docs/concepts/auth-servers/#org-authorization-server) `/authorize` endpoint. Only the org authorization server can mint access tokens that contain Okta API scopes. Refer to [Which authorization server should you use](/docs/concepts/auth-servers/#which-authorization-server-should-you-use) for more information on authorization servers and how ot use them.

Reviewer: ```suggestion
> **Important:** Request an access token from your Okta [org authorization server](/docs/concepts/auth-servers/#org-authorization-server) `/authorize` endpoint. Only the org authorization server can mint access tokens that contain Okta API scopes. Refer to [Which authorization server should you use](/docs/concepts/auth-servers/#which-authorization-server-should-you-use) for more information on authorization servers and how to use them.
```

## Create an OAuth 2.0 app in Okta

Create the client app that you want to use with the Okta APIs.

1. Sign in to your Okta organization with your administrator account.
1. In the Admin Console, go to **Applications** > **Applications**.
1. Click **Create App Integration**.
1. On the **Create a new app integration** page, select **OIDC - OpenID Connect** as the **Sign-in method**.
1. Choose **Web Application** for the **Application type**. Creating a web app is an easy way to test scope-based access to the Okta APIs using an OAuth 2.0 bearer token. Click **Next**.

>     > **Note:** It's important to choose the appropriate app type for apps that are public clients. Not choosing the right app may result in Okta API endpoints attempting to verify an app's client secret. Public clients aren't designed to have these, and so it breaks the sign-in or sign-out flow.

Reviewer: ```suggestion
    > **Note:** It's important to choose the appropriate app type for apps that are public clients. Choosing the wrong app type may result in Okta API endpoints attempting to verify an app's client secret. Because public clients aren't designed to have these, the sign-in or sign-out flow can break.
```
